test/framework/config_constants.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+const defaultConfigYAML = `---
```

Comment:
Any real reason to define this as a yaml string vs as a go struct and serialize it to yaml when needed?

Reply:
Hi @detiber,

Because that way `DefaultConfig` always ensures `LoadConfig` works.